argocd/app-of-apps/values.tmpl.yaml

```diff
@@ -110,10 +110,6 @@ argo-cd:
           - openid
           - profile
           - email
-    rbacConfig:
-      policy.default: role:readonly
-      policy.csv: |
-        g, pipeline, role:readonly
```

Comment:
This does not exist anymore in master, please rebase.

Reply:
Yeah this is a mistake. Thanks. I just rebased, and this is no longer present in the patch